Doc/library/sys.rst

```diff
@@ -1253,7 +1253,7 @@ always available.
 
    The trace function is invoked (with *event* set to ``'call'``) whenever a new
    local scope is entered; it should return a reference to a local trace
-   function to be used that scope, or ``None`` if the scope shouldn't be traced.
+   function to be used for the new scope, or ``None`` if the scope shouldn't be traced.
```

Comment:
From what I can tell, it looks like the original author intended to write this as:

> function to be used for that scope ...

Your suggestion provides an improvement though by making it a bit more explicit. 👍